README.md

```diff
@@ -8,10 +8,17 @@
 ## Getting Started
 
 - create a .env file based on .env.example and copy the content of .env.example to .env (`$ cp .env.example .env`)
-- docker compose build
-- docker compose run web bundle install
-- docker compose run web bin/rails db:setup
-- bin/dev
+- run `docker compose build`
+- run `docker compose exec web bundle install`
+- On `db/seeds.rb` uncomment the lines:
```

Comment:
having to uncomment is not cool. 
Can we safe_guard the code from the seeds for dev env only?